### DISCUSSION TITLE
how can install it in kali linux?

### DISCUSSION THREAD

**Discussion by mohamadiarch** [rafaelmardojai/blanket#228] [Q&A]
hi, is it possible to install it in kali linux?

**Comment by rafaelmardojai** [comment]
You can install flatpak (https://www.kali.org/docs/tools/flatpak/) and install the official package from Flathub (https://flathub.org/apps/details/com.rafaelmardojai.Blanket).

Alternative there is a third party package for Ubuntu that probably could be installed?  (https://launchpad.net/~apandada1/+archive/ubuntu/blanket)  I don't know if you can download the `.deb` from launchpad.

**Reply by mohamadiarch** [reply]
I just tried the first way that you said and it worked for me. flatpak successfully installed that. thanks.
